msm_scheduler/core/import_data.py

```diff
@@ -0,0 +1,139 @@
+import os.path
+import pandas as pd
+import numpy as np
+
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
+from google_auth_oauthlib.flow import InstalledAppFlow
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+
+# If modifying these scopes, delete the file token.json.
+SCOPES = ["https://www.googleapis.com/auth/spreadsheets.readonly"]
+SHEET_ID = "1B0Yq3AJXZNYVdVV0BpAFWIfRCxL17VsMrnmMxmXePmA"
+
+
+def _get_sheet_range(sheet, sheet_range):
+    result = sheet.values().get(spreadsheetId=SHEET_ID, range=sheet_range).execute()
+    values = result.get("values", [])
+    df = pd.DataFrame(values[1:], columns=values[0])
+    return df
+
+
+def import_data():
+    creds = None
+    # The file token.json stores the user's access and refresh tokens, and is
+    # created automatically when the authorization flow completes for the first
+    # time.
+    if os.path.exists("token.json"):
```

Comment:
We want to be able to import two different spreadsheets. The second spreadsheet has the same sheets and columns, but will have an addition `Base Teams` sheet. It will override values in the first if provided.

Reply:
To support this, we probably want to further abstract this method (maybe have `creds` and `columns` as parameters). E.g. `columns = ['Players!A1:F', ...]`

Reply:
Not sure what `Base Teams` is yet but I added the parameters